src/inputs/buttonConstants.js

```diff
@@ -3,24 +3,63 @@ import Utils from '../utils/utils';
 
 export const buttonPropTypes = {
     as: PropTypes.oneOf(['a', 'button']),
+    /**
+     * Primary content.
+     */
     children: PropTypes.node.isRequired,
+    /**
+     * Additional classes.
+     */
     className: PropTypes.string,
+    /**
+     * Color of the button.
+     */
     color: PropTypes.oneOf(Utils.colorEnums()),
+    /**
+     * A button can reduce its padding.
+     */
     compact: PropTypes.bool,
+    /**
+     * A button can be disabled.
+     */
     disable: PropTypes.bool,
+    /**
+     * Deprecated prop.
```

Comment:
Should we say 'Deprecated prop.  Please use \`disable\` instead.'?

Reply:
Fixed.

Reply:
Update in 170fcf6 looks good.  Resolving.